pythonforandroid/recipes/zope_interface/__init__.py

```diff
@@ -9,10 +10,18 @@ class ZopeInterfaceRecipe(PythonRecipe):
     version = '4.1.3'
     url = 'https://pypi.python.org/packages/source/z/zope.interface/zope.interface-{version}.tar.gz'
     site_packages_name = 'zope.interface'
-
-    depends = [('python2', 'python3crystax')]
+    depends = ['setuptools']
     patches = ['no_tests.patch']
 
+    def build_arch(self, arch):
+        super(ZopeInterfaceRecipe, self).build_arch(arch)
+        # The zope.interface module lacks of the __init__.py file in one of his
+        # folders (once is installed), that leads into an ImportError.
+        # Here we intentionally apply a patch to solve that, so, in case that
+        # this is solved in the future an error will be triggered
+        zope_install = join(self.ctx.get_site_packages_dir(arch.arch), 'zope')
+        self.apply_patch('fix-init.patch', arch.arch, build_dir=zope_install)
```

Comment:
Minor: I'm not sure I understand why we're applying the patch that way rather than adding it to the `patches` list?

Reply:
ahah fun, thanks for the explanation!
Then is it not the manifest file or something like that we want to fix with a patch?
I mean the way you did seems fine, I'm just challenging to see if there're no better ways